sys/include/net/zep.h

```diff
@@ -9,6 +9,7 @@
 /**
  * @defgroup    net_zep  ZEP header definitions
  * @ingroup     net
+ * @brief       ZigBee Encapsulation Protocol
```

Comment:
This is not what this file is ;-). Maybe "Definitions for the ZigBee Encapsulation Protocol", but that is quite redundant to the title (exapt for the writing out of the abbreviation)

Reply:
You are right, I changed this and added cross references between `net_zep` and `drivers_socket_zep` groups.